support-frontend/assets/components/stripeCardForm/stripePaymentButton.tsx

```diff
@@ -37,6 +38,21 @@ export function StripePaymentButton(): JSX.Element {
 		(state) => state.page.checkoutForm.payment.stripe,
 	);
 
+	const payWithStripe = useFormValidation(function pay() {
```

Comment:
I know this works, but it seems unnecessary to pass a named function declaration as a parameter here, this could simply be an anonymous arrow function, and that feels the typical way we'd do this? Just interested if there's any reason we'd do it as you've done?

Reply:
I can see a case that using a named function is more readable, eg. use form validate then pay.

Reply:
Named functions create much more readable stack traces. Because the actual execution of the payment function is deferred until after validation, it'll be a bit clearer what's happening if the payment function throws an error.